Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -791,7 +792,7 @@ u32 PPCAnalyzer::Analyze(u32 address, CodeBlock* block, CodeBuffer* buffer, std:
       }
     }
 
-    if (follow)
+    if (follow && Config::Get(Config::MAIN_JIT_FOLLOW_BRANCH))
```

Comment:
Instead of putting the check here, could you put it on the line where we check `HasOption(OPTION_BRANCH_FOLLOW)`? That way, we can skip executing some code when the branch following option is off.